prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -77,8 +85,8 @@ Component "{{ .Env.XMPP_MUC_DOMAIN }}" "muc"
         {{ if .Env.XMPP_MUC_MODULES }}
         "{{ join "\";\n\"" (splitList "," .Env.XMPP_MUC_MODULES) }}";
         {{ end }}
-        {{ if eq $AUTH_TYPE "jwt" }}
-        "token_verification";
+        {{ if .Env.JWT_ENABLE_TOKEN_AUTH | default "0" | toBool }}
```

Comment:
this check is incorrect, `JWT_ENABLE_TOKEN_AUTH` no longer exists

Reply:
resolved, good catch.